Formula/handbrake.rb

```diff
@@ -4,7 +4,7 @@ class Handbrake < Formula
   url "https://github.com/HandBrake/HandBrake/releases/download/1.4.2/HandBrake-1.4.2-source.tar.bz2"
   sha256 "8b8e81b7dc2e3180f4e94e8c7f5337d2953f69f0d983ccce48096e29ed6dfb61"
   license "GPL-2.0-only"
-  revision 1
+  revision 2
```

Comment:
```suggestion
  revision 1
```
Python is a `:build` only dependency, so let's omit the `revision` bump, please.